tests/e2e/features/createSpace.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Create a space
+    As a super user
+    I want to create a space
+    So that I can add recipes and share my recipes
+
+
+    Background:
```

Comment:
extra one new line is specially for the tags. But `Background` cannot have any tag
```suggestion
    Background:
```